docs/react/recommendation.zh-CN.md

```diff
@@ -18,7 +18,7 @@ title: 社区精选组件
 | 响应式 | [react-responsive](https://github.com/contra/react-responsive) [react-media](https://github.com/ReactTraining/react-media) |
 | 复制到剪贴板 | [react-copy-to-clipboard](https://github.com/nkbt/react-copy-to-clipboard) |
 | 页面 meta 属性 | [react-helmet](https://github.com/nfl/react-helmet) [react-helmet-async](https://github.com/staylor/react-helmet-async) |
-| 图标 | [react-fa](https://github.com/andreypopp/react-fa) [react-icons](https://github.com/gorangajic/react-icons) |
+| 图标 | [react-fa](https://github.com/FortAwesome/react-fontawesome) [react-icons](https://github.com/gorangajic/react-icons) |
```

Comment:
should be `[react-fontawesome](https://github.com/FortAwesome/react-fontawesome)`

Reply:
😂

Reply:
Thanks! I forgot about that.